RPerformanceTracking/Private/UIDevice+RPerformanceTracking.m

```diff
@@ -0,0 +1,49 @@
+#import "UIDevice+RPerformanceTracking.h"
+#import <mach/mach.h>
+
+@implementation UIDevice (RPerformanceTracking)
+
+- (int64_t)freeDeviceMemory
+{
+    vm_size_t pagesize;
+
+    mach_port_t host_port = mach_host_self();
+    mach_msg_type_number_t host_size = HOST_VM_INFO64_COUNT;
+    host_page_size(host_port, &pagesize);
+
+    vm_statistics64_data_t vmstat;
+
+    if (host_statistics64(host_port, HOST_VM_INFO64, (host_info64_t)&vmstat, &host_size) != KERN_SUCCESS)
+        return -1;
+
+    uint64_t free_mem_in_bytes = vmstat.free_count * pagesize;
+    return (long long)(free_mem_in_bytes / 1024);
```

Comment:
What's the memory unit returned in these methods? It seems like KB. I think MB is accurate enough and we should be in sync with the android implementation.